community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/TransactionIdStore.java

```diff
@@ -43,16 +43,47 @@
  */
 public interface TransactionIdStore
 {
+    // TODO: Document this somewhere.
```

Comment:
You can complete it and add it as an implementation note in the javadoc for this interface.